.github/workflows/create-release.yml

```diff
@@ -56,6 +56,8 @@ jobs:
         prerelease: false
 
     - name: Publish Pods
+      env:
+        COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
```

Comment:
Can you explain how `COCOAPODS_TRUNK_TOKEN` is used and where it was obtained from?

Reply:
The token was obtained by running `pod trunk register "<email>" "name"` and validating the session request.

Afterwards, I copied the token by running `grep -A2 'trunk.cocoapods.org' ~/.netrc` and copied the resulting `password`

```
machine trunk.cocoapods.org
  login <email-address>
  password <token-value>

```

Reply:
I see, was that under your email address or the iOS one?